_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,137 @@
+
+Apart from the language improvements mentioned above, some other interesting localization changes happend in this release. [@comradekingu](https://github.com/comradekingu), a frequent contributor to and language checker of NewPipe and other Open Source projects, has once again taken a close look at our strings. There was, as always, a lot of room for improvement. Our wonderful translators rushed to help and immediately implemented his suggestions.
+
+We are also happy to provide **new localizations** in this release: Sardinian, Bengali, Javanese and Berber
+
+If you are missing a translation or think a phrase needs adjustmemnt, head over to [Weblate](https://hosted.weblate.org/engage/newpipe/) and help us to improve NewPip's translation even more!
+
+## Fixes
+
+If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+
+An empty error report in certain kinds of crashes is not empty anymore, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code! Stypox also fixed a crash when NewPipe faced empty comments, now it will display it just as empty as it is.
+
+Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab, whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
+
+The license pop-up doesn't disappear when you change orientation now. This was introduced by [@nmurali94](https://github.com/nmurali94). And [@budde92](https://github.com/budde25) ensured that you can now delete a file you just downloaded and then try to re-download it without the app crashing. We are not sure why you'd want to do this, but now you can do so without any worries.
+
+[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Yay for less redundancy. When auto-queue is enabled, Newpipe wants to add a stream to the queue the moment you open a video. But if there are no video suggestions on that page, it causes a crash. This has been fixed by [@wb9688](https://github.com/wb9688).
+
+## Development
+
+Development. Where all the magic which keeps this app running happens. It saw some improvements in this release: (Warning: Nerd talk ahead)
+
+- A Checkstyle rule was added so it would show an error when a local variable which could be `final` wasn't declared as such. ([@XiangRongLin](https://github.com/XiangRongLin))
+
+- The extractor will ignore OTF streams now, since they don't work with progressive HTTP and the extractor can't work with DASH just yet ([@wb9688](https://github.com/wb9688))
+
+- PrettyTime is updated to 4.0.6, so a workaround can be removed from NewPipe's code and a perfomance improvement is part of it now ([@B0pol](https://github.com/B0pol)
```

Comment:
```suggestion
- PrettyTime is updated to 4.0.6, so a workaround can be removed from NewPipe's code and a perfomance improvement is part of it now ([@B0pol](https://github.com/B0pol))
```

Reply:
This is for users, you should say what problems the update fixes, not that a workaround could be removed from our code.

Reply:
Plural for days, used in relative date (e.g. 4 days ago), is fixed in russian with this prettytime update